src/main/java/az/edu/turing/games/weekplanner/TaskManager.java

```diff
@@ -0,0 +1,47 @@
+package az.edu.turing.games.weekplanner;
+
+import java.util.Optional;
+import java.util.Scanner;
+
+public class TaskManager {
+    private WeekPlanner planner;
+    private Scanner sc;
+
+    public TaskManager() {
+        planner = new WeekPlanner();
+        sc = new Scanner(System.in);
+    }
+
+    public void run() {
+        while (true) {
+            System.out.print("please input the day of the week(or exit to quit):");
+            String input = sc.nextLine().trim();
+            if (input.equalsIgnoreCase("exit")) {
+                System.out.println("the program has ended ");
+                break;
+            } else if (input.startsWith("change")) {
+                handleTaskChange(input);
+            } else {
+                showTasks(input);
+            }
+        }
+    }
+
+    private void handleTaskChange(String input) {
+        String dayToChange = input.substring(input.indexOf(' ') + 1).trim();
```

Comment:
you can use .split here for taking the day

Reply:
done✅